### PR TITLE
sprint-4() : merge sprint-4 developments

### PR DESCRIPTION
**Pull request by ThomRick**

### CODE REVIEW

File: src/templates/ts/application/package.json [hunk 1]
```diff
@@ -1,61 +1,60 @@
...
+    "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "webpack": "webpack --config webpack.config.js"
   },
   "dependencies": {
-    "@nestjs/common": "^5.0.0",
-    "@nestjs/core": "^5.0.0", <% if (dependencies.indexOf('microservices') !== -1) { %> 
-    "@nestjs/microservices": "^5.0.0", <% } %><% if (dependencies.indexOf('websockets') !== -1) { %> 
-    "@nestjs/websockets": "^5.0.0", <% } %>
```
Comment (kamilmysliwiec) [inline src/templates/ts/application/package.json]: not sure if we should remove it^

File: src/templates/ts/application/README.md [hunk 2]
```diff
@@ -1,38 +1,79 @@
...
 ## Description
 
-<%= description %>
+[Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
 
 ## Installation
 
 ```bash
-$ <%= packageManager %> install
+$ npm install
```
Comment (kamilmysliwiec) [inline src/templates/ts/application/README.md]: not sure if we should remove it^

File: src/templates/ts/application/package.json (hunk 1 above)
Comment (ThomRick) [reply src/templates/ts/application/package.json]: if we leave it we can start common and core only.
I think we don't need microservices and websockets for an frontend rest api that calls services and aggregate data for a frontend web client, don't you ?

File: src/templates/ts/application/README.md (hunk 2 above)
Comment (ThomRick) [reply src/templates/ts/application/README.md]: it can set yarn if user use yarn package manager

Comment (kamilmysliwiec) [reply src/templates/ts/application/README.md]: that's why I'm asking

Comment (kamilmysliwiec) [reply src/templates/ts/application/README.md]: :D

File: src/templates/ts/application/package.json (hunk 1 above)
Comment (kamilmysliwiec) [reply src/templates/ts/application/package.json]: same as below, not sure why it's removed

**Pull request merged**